src/components/ArticleDetail.jsx

```diff
@@ -1,22 +1,46 @@
-const ArticleDetail= ({articles}) => {
+import ShowCommentsByArticle from "./ShowCommentsByArticle";
+import { useState } from "react";
+
+const ArticleDetail= ({article}) => {
+
+  const [article_id,setArticleId] = useState("");
+
+   const handleFormViewComments = (e) => {
+       e.preventDefault();
+   }
+
+    const handleOnClickViewComments = (e) => {
+     
+      if (e.target.textContent === 'View Comments') {
+         e.target.textContent = 'Un View';
+         setArticleId(article.article_id)
+      } else {
+         e.target.textContent = 'View Comments';
+         setArticleId("")
+      }
+      
+    }
+
     return (
-        <div>
-            {
-                 articles.map((article) =>{
-                 return  ( 
-                       <div key = {article.article_id}>
-                         <header name="Title">Title: {article.title}</header>
-                         <header name="Author">Author: {article.author}</header>
-                         <header name="Topic">Topic: {article.topic}</header>
-                         <header name ="Created Date">Created_at: {article.created_at.substring(0,10)}</header>
-                         <header name = "Comment Count">Comment Count: {article.comment_count}</header>
-                         <header name = "Vote">Votes: {article.votes}</header>
-                        <img src = {article.article_img_url}  alt={article.title} />
-                      </div>
-                 )
-               })
-            } 
-            
+        <div> 
+         <form>
+              <header name="Title">Title: {article.title}</header>
+              <header name="Author">Author: {article.author}</header>
+              <header name="Topic">Topic: {article.topic}</header>
+              <header name="Body">Body: {article.body}</header>
+              <header name ="Created Date">Created Date: {article.created_at.substring(0,10)}</header>
+              <header name = "Comment Count">Comment Count: {article.comment_count}</header>
+              <header name = "Vote">Votes: {article.votes}</header>
+              <img src = {article.article_img_url}  alt={article.title} />
+          </form> 
```

Comment:
these no longer need to be wrapped in a <form> as there is no form here - is there a better tag we can wrap them in?  `<main> <section>` etc